<html>
<head>
<title>TestNG:  Add And Edit All Masters3</title>
<link href="../testng.css" rel="stylesheet" type="text/css" />
<link href="../my-testng.css" rel="stylesheet" type="text/css" />

<style type="text/css">
.log { display: none;} 
.stack-trace { display: none;} 
</style>
<script type="text/javascript">
<!--
function flip(e) {
  current = e.style.display;
  if (current == 'block') {
    e.style.display = 'none';
    return 0;
  }
  else {
    e.style.display = 'block';
    return 1;
  }
}

function toggleBox(szDivId, elem, msg1, msg2)
{
  var res = -1;  if (document.getElementById) {
    res = flip(document.getElementById(szDivId));
  }
  else if (document.all) {
    // this is the way old msie versions work
    res = flip(document.all[szDivId]);
  }
  if(elem) {
    if(res == 0) elem.innerHTML = msg1; else elem.innerHTML = msg2;
  }

}

function toggleAllBoxes() {
  if (document.getElementsByTagName) {
    d = document.getElementsByTagName('div');
    for (i = 0; i < d.length; i++) {
      if (d[i].className == 'log') {
        flip(d[i]);
      }
    }
  }
}

// -->
</script>

</head>
<body>
<h2 align='center'>Add And Edit All Masters3</h2><table border='1' align="center">
<tr>
<td>Tests passed/Failed/Skipped:</td><td>5/5/0</td>
</tr><tr>
<td>Started on:</td><td>Mon Sep 16 18:05:52 IST 2024</td>
</tr>
<tr><td>Total time:</td><td>137 seconds (137798 ms)</td>
</tr><tr>
<td>Included groups:</td><td></td>
</tr><tr>
<td>Excluded groups:</td><td></td>
</tr>
</table><p/>
<small><i>(Hover the method name to see the test class name)</i></small><p/>
<table width='100%' border='1' class='invocation-failed'>
<tr><td colspan='4' align='center'><b>FAILED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='tests.AllMasters.VerifyUserIsAbleToAddDriver()'><b>VerifyUserIsAbleToAddDriver</b><br>Test class: tests.AllMasters</td>
<td><div><pre>java.lang.AssertionError: Expected condition failed: waiting for visibility of element located by By.xpath: //div[@id=&apos;toast-container&apos;] (tried for 30 second(s) with 500 milliseconds interval)
Build info: version: &apos;4.21.0&apos;, revision: &apos;79ed462ef4&apos;
System info: os.name: &apos;Windows 10&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;21&apos;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 128.0.6613.138, chrome: {chromedriverVersion: 128.0.6613.137 (fe621c5aa2d..., userDataDir: C:\Windows\SystemTemp\scope...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:54303}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:54303/devtoo..., se:cdpVersion: 128.0.6613.138, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 6d6231c004dbfc911bd02f630e7ec3b6
	at web.pages.BaseMethods.waitForElementVisibility(BaseMethods.java:152)
	at web.pages.BaseMethods.getElementText(BaseMethods.java:231)
	at web.pages.BasePage.getTostMessage(BasePage.java:80)
	at tests.AllMasters.VerifyUserIsAbleToAddDriver(AllMasters.java:58)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 17 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1002366686", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1002366686'><pre>java.lang.AssertionError: Expected condition failed: waiting for visibility of element located by By.xpath: //div[@id=&apos;toast-container&apos;] (tried for 30 second(s) with 500 milliseconds interval)
Build info: version: &apos;4.21.0&apos;, revision: &apos;79ed462ef4&apos;
System info: os.name: &apos;Windows 10&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;21&apos;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 128.0.6613.138, chrome: {chromedriverVersion: 128.0.6613.137 (fe621c5aa2d..., userDataDir: C:\Windows\SystemTemp\scope...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:54303}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:54303/devtoo..., se:cdpVersion: 128.0.6613.138, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 6d6231c004dbfc911bd02f630e7ec3b6
	at org.testng.Assert.fail(Assert.java:110)
	at web.pages.BaseMethods.waitForElementVisibility(BaseMethods.java:152)
	at web.pages.BaseMethods.getElementText(BaseMethods.java:231)
	at web.pages.BasePage.getTostMessage(BasePage.java:80)
	at tests.AllMasters.VerifyUserIsAbleToAddDriver(AllMasters.java:58)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at org.testng.TestRunner.privateRun(TestRunner.java:808)
	at org.testng.TestRunner.run(TestRunner.java:603)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:32)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:467)
	at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
</pre></div></td>
<td>56</td>
<td>tests.AllMasters@10289886</td></tr>
<tr>
<td title='tests.AllMasters.VerifyUserIsAbleToAddTruck()'><b>VerifyUserIsAbleToAddTruck</b><br>Test class: tests.AllMasters</td>
<td><div><pre>java.lang.AssertionError: The following asserts failed:
	Message do not match expected [Truck added successfully] but found [Truck already exists with this unit number]
	at tests.AllMasters.VerifyUserIsAbleToAddTruck(AllMasters.java:76)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 18 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1285827047", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1285827047'><pre>java.lang.AssertionError: The following asserts failed:
	Message do not match expected [Truck added successfully] but found [Truck already exists with this unit number]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
	at tests.AllMasters.VerifyUserIsAbleToAddTruck(AllMasters.java:76)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at org.testng.TestRunner.privateRun(TestRunner.java:808)
	at org.testng.TestRunner.run(TestRunner.java:603)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:32)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:467)
	at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
</pre></div></td>
<td>15</td>
<td>tests.AllMasters@10289886</td></tr>
<tr>
<td title='tests.AllMasters.VerifyUserIsAbleToAddTrailer()'><b>VerifyUserIsAbleToAddTrailer</b><br>Test class: tests.AllMasters</td>
<td><div><pre>java.lang.AssertionError: The following asserts failed:
	Message do not match expected [Truck added successfully] but found [Truck already exists with this unit number],
	Message do not match expected [Trailer added successfully] but found [Trailer already exists with this unit number]
	at tests.AllMasters.VerifyUserIsAbleToAddTrailer(AllMasters.java:93)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 18 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1200592549", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1200592549'><pre>java.lang.AssertionError: The following asserts failed:
	Message do not match expected [Truck added successfully] but found [Truck already exists with this unit number],
	Message do not match expected [Trailer added successfully] but found [Trailer already exists with this unit number]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
	at tests.AllMasters.VerifyUserIsAbleToAddTrailer(AllMasters.java:93)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at org.testng.TestRunner.privateRun(TestRunner.java:808)
	at org.testng.TestRunner.run(TestRunner.java:603)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:32)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:467)
	at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
</pre></div></td>
<td>13</td>
<td>tests.AllMasters@10289886</td></tr>
<tr>
<td title='tests.AllMasters.VerifyUserIsAbleToAddCustomer()'><b>VerifyUserIsAbleToAddCustomer</b><br>Test class: tests.AllMasters</td>
<td><div><pre>java.lang.AssertionError: The following asserts failed:
	Message do not match expected [Truck added successfully] but found [Truck already exists with this unit number],
	Message do not match expected [Trailer added successfully] but found [Trailer already exists with this unit number],
	Message do not match expected [Customer added successfully] but found [Customer already exists with this email]
	at tests.AllMasters.VerifyUserIsAbleToAddCustomer(AllMasters.java:113)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 18 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1055599190", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1055599190'><pre>java.lang.AssertionError: The following asserts failed:
	Message do not match expected [Truck added successfully] but found [Truck already exists with this unit number],
	Message do not match expected [Trailer added successfully] but found [Trailer already exists with this unit number],
	Message do not match expected [Customer added successfully] but found [Customer already exists with this email]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
	at tests.AllMasters.VerifyUserIsAbleToAddCustomer(AllMasters.java:113)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at org.testng.TestRunner.privateRun(TestRunner.java:808)
	at org.testng.TestRunner.run(TestRunner.java:603)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:32)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:467)
	at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
</pre></div></td>
<td>10</td>
<td>tests.AllMasters@10289886</td></tr>
<tr>
<td title='tests.AllMasters.VerifyUserIsAbleToAddVendor()'><b>VerifyUserIsAbleToAddVendor</b><br>Test class: tests.AllMasters</td>
<td><div><pre>java.lang.AssertionError: The following asserts failed:
	Message do not match expected [Truck added successfully] but found [Truck already exists with this unit number],
	Message do not match expected [Trailer added successfully] but found [Trailer already exists with this unit number],
	Message do not match expected [Customer added successfully] but found [Customer already exists with this email],
	Message do not match expected [Vendor added successfully] but found [Vendor already exists with this email]
	at tests.AllMasters.VerifyUserIsAbleToAddVendor(AllMasters.java:132)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 18 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace430264104", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace430264104'><pre>java.lang.AssertionError: The following asserts failed:
	Message do not match expected [Truck added successfully] but found [Truck already exists with this unit number],
	Message do not match expected [Trailer added successfully] but found [Trailer already exists with this unit number],
	Message do not match expected [Customer added successfully] but found [Customer already exists with this email],
	Message do not match expected [Vendor added successfully] but found [Vendor already exists with this email]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
	at tests.AllMasters.VerifyUserIsAbleToAddVendor(AllMasters.java:132)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at org.testng.TestRunner.privateRun(TestRunner.java:808)
	at org.testng.TestRunner.run(TestRunner.java:603)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:32)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:467)
	at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
</pre></div></td>
<td>8</td>
<td>tests.AllMasters@10289886</td></tr>
</table><p>
<table width='100%' border='1' class='invocation-passed'>
<tr><td colspan='4' align='center'><b>PASSED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='tests.AllMasters.DashboardVendorList()'><b>DashboardVendorList</b><br>Test class: tests.AllMasters</td>
<td></td>
<td>3</td>
<td>tests.AllMasters@10289886</td></tr>
<tr>
<td title='tests.AllMasters.DashboardTruckList()'><b>DashboardTruckList</b><br>Test class: tests.AllMasters</td>
<td></td>
<td>3</td>
<td>tests.AllMasters@10289886</td></tr>
<tr>
<td title='tests.AllMasters.DashboardCustomerList()'><b>DashboardCustomerList</b><br>Test class: tests.AllMasters</td>
<td></td>
<td>3</td>
<td>tests.AllMasters@10289886</td></tr>
<tr>
<td title='tests.AllMasters.DashboardDriverList()'><b>DashboardDriverList</b><br>Test class: tests.AllMasters</td>
<td></td>
<td>2</td>
<td>tests.AllMasters@10289886</td></tr>
<tr>
<td title='tests.AllMasters.DashboardTrailerList()'><b>DashboardTrailerList</b><br>Test class: tests.AllMasters</td>
<td></td>
<td>3</td>
<td>tests.AllMasters@10289886</td></tr>
</table><p>
</body>
</html>